.circleci/config.yml

```diff
@@ -0,0 +1,170 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/coala-bears
+
+    docker:
+      - image: circleci/jdk8:0.1.1
+
+    parallelism: 2
+
+    steps:
+      - checkout
+      - restore_cache:
+          key: dependency-cache-{{checksum "requirements.txt"}}
+      - run:
+          name: dependencies
+          command: |
+                    sudo apt-get update
+                    sudo apt-get install -y software-properties-common python-software-properties curl
+                    sudo add-apt-repository -y ppa:hvr/ghc
+                    sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test
+                    sudo add-apt-repository -y ppa:avsm/ppa
+                    curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash  -
+                    curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
+                    echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
+                    sudo apt-get update
+                    sudo apt-get install -y python3.4-venv git bzip2 \
+                      cabal-install-1.24 clang-3.4 libreadline-dev libyaml-dev\
+                      gfortran ghc happy indent libblas-dev liblapack-dev \
+                      libperl-critic-perl libxml2-utils menhir php-codesniffer \
+                      build-essential ruby texinfo libbz2-dev libcurl4-openssl-dev \
+                      libexpat-dev libncurses-dev zlib1g-dev git-core libssl-dev \
+                      libsqlite3-dev sqlite3 libxslt1-dev libffi-dev yarn python3-dev \
+                      libgdbm-dev libncurses5-dev automake libtool bison
+      - run:
```

Comment:
One blank here too

Reply:
Correct all indentation and blank line before `- run`

Reply:
Sure sangam, I will correct all the indentation in the next push
Thanks